### ISSUE TITLE
#

### ISSUE THREAD

**Issue by myGistSnippets** [palantir/tslint#1858]
### Bug Report

- __TSLint version__: `<fill this out>`
- __TypeScript version__: `<fill this out>`
- __Running TSLint via__: (please choose one) CLI / Node.js API / grunt-tslint / Atom / Visual Studio / etc

#### TypeScript code being linted

```ts
(include if relevant)
```

with `tslint.json` configuration:

```json
(include if relevant)
```

#### Actual behavior

(fill this out)

#### Expected behavior

(fill this out)

**Comment by JoshuaKGoldberg** [comment]
wat
